config/routes.rb

```diff
@@ -1,7 +1,14 @@
 Rails.application.routes.draw do
-  resources :users
+  # resources :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/users", to: "users#index", as: :users
+  get "/users/:id", to: "users#show", as: :user
+  get "/users/:user_id/posts", to: "posts#index", as: :user_posts
+  get "/users/:user_id/posts/:id", to: "posts#show", as: :user_post
+
```

Comment:
- [ ] Great work configuring your routes. However, as much as configuring your routes like this works, it is not the most efficient way to do it. Just imagine you are working on a large application with a lot of routes, configuring your routes like this will quickly become tedious and unmaintainable and that is why rails have a better way of configuring your routes. To learn more about this kindy check this [article](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router)

Kindly try and configure your routes well.

### HINT

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
@peterrobert Thank you very much for your review on this PR and taking me to right direction. I have added `show` action in both the controllers and refactored method in `config/routes.rb` file

Thank You.

![image](https://user-images.githubusercontent.com/40719899/204212981-a7315485-be2e-4b1f-b3f8-a6bff7f1903f.png)